mteb/abstasks/Audio/AbsTaskZeroshotAudioClassification.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from datasets import Dataset
+
+from mteb.abstasks.TaskMetadata import HFSubset
+
+from ...encoder_interface import Encoder
+from ..AbsTask import AbsTask, ScoresDict
+from ..evaluation.evaluators import ZeroshotAudioClassificationEvaluator
+
+logger = logging.getLogger(__name__)
+
+
+class AbsTaskZeroshotAudioClassification(AbsTask):
```

Comment:
I think you're calculating similarity between these embeddings and it's better to name like
```suggestion
class AbsTaskTextAudioPairClassification(AbsTask):
```